maintainers/maintainer-list.nix

```diff
@@ -2613,6 +2613,12 @@
     githubId = 231483;
     name = "Jack Kelly";
   };
+  enderger = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add enderger``.